testing.py

```diff
@@ -1391,3 +1396,9 @@ def text(self):
         if isinstance(self.content, bytes):
             return self.content.decode("utf8")
         return self.content
+
+    def raise_for_status(self):
```

Comment:
So this function is needed to make the mock request for realistic?

Reply:
Exactly. sudz thinks it has a real Requests response and calls this method on it, but it actually has a mocked response.